src/main/java/pl/javastart/task/Mix.java

```diff
@@ -0,0 +1,122 @@
+package pl.javastart.task;
+
+public class Mix extends Agreement {
+
+    private int smsLimit;
+    private int mmsLimit;
+    private int secLimit;
+    private double smsPrice;
+    private double mmsPrice;
+    private double perMinPrice;
+
+    public Mix(double accountState, int smsLimit, int mmsLimit, int secLimit, double smsPrice, double mmsPrice, double perMinPrice) {
+        super(accountState);
+        this.smsLimit = smsLimit;
+        this.mmsLimit = mmsLimit;
+        this.secLimit = secLimit;
+        this.smsPrice = smsPrice;
+        this.mmsPrice = mmsPrice;
+        this.perMinPrice = perMinPrice;
+    }
+
+    @Override
+    public void sendSms() {
+
+        int smsCount = getSmsCount();
+        if (smsLimit > 0) {
+            System.out.println("SMS wysłany\n");
+            smsLimit--;
+            setSmsCount(smsCount + 1);
+            return;
+        }
+        double accountState = getAccountState();
+        if (accountState > smsPrice) {
+            System.out.println("SMS wysłany\n");
+            setSmsCount(smsCount + 1);
+            setAccountState(accountState - smsPrice);
+            return;
+        }
+        System.out.println("Brak środków na koncie\n");
+    }
+
```

Comment:
Jeżeli będziesz dziedziczył po klasie Prepaid, to zarówno w tej metodzie, jak i pozostałych, będziesz mógł wykorzystać część logiki z klasy Prepaid, wywołując metody przez `super.nazwaMetody()`